layer/executables/packager.py

```diff
@@ -11,7 +11,9 @@
 from dataclasses import dataclass, field
 from pathlib import Path
 from typing import Any, Callable, Dict, Generator, Optional, Sequence, Tuple
+from xmlrpc.client import Boolean
```

Comment:
is this the source for the `Boolean` type that we want? xmlrpc looks odd.

Reply:
Silly auto import, all I needed was just `bool`. Fixed. Thank you!